src/components/audio-player/helper-icons/helper-icons.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I feel like if this is in `/src` it should maybe be individual `.svg` files and we use a webpack loader to import them?

But if the intention is that these aren't "part of Marble," we can just move this file to `.storybook/helpers` and keep it as JS strings

Reply:
Yeah I was thinking these aren't really "part of marble". I'm down to move them to storybook helpers.

Reply:
Should be fixed by https://github.com/metmuseum/marble/pull/131/commits/b239c71d22c2636724607a07441075178f78b1b1